app/Models/User.php

```diff
@@ -79,6 +79,10 @@ public static function firstOrCreateByAttributes($attributes)
         );
     }
 
+    public function hasToken(){
+        return $this->tokens->count() ? 'Yes' : 'No';
```

Comment:
meh... let's make it a bool and also check that the token isn't expired or valid.

Reply:
maybe currentAccessToken() ?